core/io.c

```diff
@@ -1018,27 +1017,74 @@ our_memcpy_vs_libc(void)
     memset(src, -1, alloc_size);
     memset(dst, 0, alloc_size);
 
-    our_memcpy_start = query_time_millis();
+    int tests_size[] = { 1, 4, 128, 512, 8192, alloc_size };
+    int j;
+    for (j = 0; j < sizeof(tests_size) / sizeof(int); j++) {
+        our_memcpy_start = query_time_millis();
+        for (i = 0; i < loop_count; i++) {
+            memcpy(src, dst, tests_size[j]);
+        }
+        our_memcpy_end = query_time_millis();
+
+        libc_memcpy_start = query_time_millis();
+        for (i = 0; i < loop_count; i++) {
+            glibc_memcpy(src, dst, tests_size[j]);
+        }
+        libc_memcpy_end = query_time_millis();
+
+        our_memcpy_time = our_memcpy_end - our_memcpy_start;
+        libc_memcpy_time = libc_memcpy_end - libc_memcpy_start;
+        print_file(STDERR,
+                   "our_memcpy_time: size=" UINT64_FORMAT_STRING " " UINT64_FORMAT_STRING
+                   "\n",
+                   tests_size[j], our_memcpy_time);
+        print_file(STDERR,
+                   "libc_memcpy_time: size=" UINT64_FORMAT_STRING " " UINT64_FORMAT_STRING
```

Comment:
Please add the printed results for this and the memset times to the PR description.